### PR TITLE
Define and activate release v13.1.0 for kvm

### PR DESCRIPTION
**Pull request by yulianedyalkova**

### CODE REVIEW

File: kvm/v13.1.0/release.diff [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  - name: etcd                                                       - name: etcd
+    version: 3.4.13                                                    version: 3.4.13
+  - name: flannel-operator                                           - name: flannel-operator
+    version: 0.2.0                                                     version: 0.2.0
+  - name: kubernetes                                                 - name: kubernetes
+    version: 1.18.12                                                   version: 1.18.12
+  - name: kvm-operator                                               - name: kvm-operator
+    releaseOperatorDeploy: true                                        releaseOperatorDeploy: true
+    version: 3.14.1                                             |      version: 3.14.2
+  date: "2021-02-23T12:00:00Z"                                  |    date: "2021-02-25T15:25:30Z"
```
Comment (tfussell) [inline kvm/v13.1.0/release.diff:54]:
```suggestion
  date: "2021-02-23T12:00:00Z"                                  |    date: "2021-03-04T15:25:30Z"
```

File: kvm/v13.1.0/release.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  - name: etcd
+    version: 3.4.13
+  - name: flannel-operator
+    version: 0.2.0
+  - name: kubernetes
+    version: 1.18.12
+  - name: kvm-operator
+    releaseOperatorDeploy: true
+    version: 3.14.2
+  date: "2021-02-25T15:25:30Z"
```
Comment (tfussell) [inline kvm/v13.1.0/release.yaml:52]:
```suggestion
  date: "2021-03-03T15:25:30Z"
```

**Comment by tfussell** [comment]
We might want to deprecate 13.0.1 in this PR since the only difference between 13.0.1 and 13.1.0 is the QEMU version. What does @giantswarm/team-rocket think?

**Comment by yulianedyalkova** [comment]
There is also the [request](https://github.com/giantswarm/releases/pull/625) to bump more components:
*  `app-operator@3.2.0`
* `chart-operator 2.9.0`
* `cluster-operator@0.24.2`
* `cert-operator@1.0.1`

@giantswarm/team-rocket  do we want to include them in this release or do we want to ship qemu only first?

**Comment by yulianedyalkova** [comment]
Just found [this](https://github.com/giantswarm/releases/pull/620#issuecomment-784243841) and talked to @cornelius-keller  and we decided to go with the bump. We need to validate if it will be possible for `puma` to directly upgrade from `12.3.2` to `13.1.0` though or we have to update to `13.0.1` first.

**Comment by yulianedyalkova** [comment]
Tested the upgrade from `12.3.3` to `13.1.0` and it was successful so I will deprecate `13.0.1`. cncf and some reviews are all that we need to get this merged.

**Comment by yulianedyalkova** [comment]
@giantswarm/team-rocket @giantswarm/sig-product can I please get some reviews? Customers are asking for the release :)

**Pull request merged**